app/jobs/spotlight_search/export_job.rb

```diff
@@ -0,0 +1,33 @@
+module SpotlightSearch
+  class ExportJob < ApplicationJob
+    def perform(email, records, columns = nil, filters = nil)
+      # data = get_data(records.class, filters)
+      file_path = create_excel(records.as_json, records.class.name)
+      subject = "#{records.class.name&.humanize} export at #{Time.now}"
+      ExportMailer.send_excel_file(email, file_path, subject).deliver_now
+    end
```

Comment:
@sajinmp - can we also delete the file once it is sent? In the long run, it will come in handy

Reply:
did that. missed it